### ISSUE TITLE
Mutation of CustomPost Types - setting of Custom Taxonomies Within Custom Post Type - does not work

### ISSUE THREAD

**Issue by florianschommertz** [wp-graphql/wp-graphql#960]
I tried to set a CustomTaxonomy via WP-GraphQL on a Custom Post Type.
The 95% same syntax works with posts and a normal Wordpress category.

Themefiles for Custom Post Types/Taxonomies
[https://github.com/florianschommertz/wp-graphql-mutation-problem](url)
Please check Screenshot

<img width="1689" alt="Screen Shot 2019-09-19 at 23 25 02" src="https://user-images.githubusercontent.com/332235/65282622-a042e900-db35-11e9-92a4-6fa6621b59a2.png">

Code:

```
mutation MutateCustomPostWithCustomTaxonomy {
  updateQlHolidayhome(input: {clientMutationId: "****", id: "aG9saWRheWhvbWVzOjE0Nw==", qlAmenities: {nodes: {id: "YW1lbml0aWVzOjI5"}, append: true}}) {
    qlHolidayhome {
      qlAmenities {
        edges {
          node {
            id
            name
          }
        }
      }
    }
  }
}

mutation MutatePostWithCategory {
  updatePost(input: {clientMutationId: "****", id: "cG9zdDoxNDU=", categories: {append: true, nodes: {id: "Y2F0ZWdvcnk6MQ=="}}}) {
    post {
      categories {
        edges {
          node {
            id
            name
          }
        }
      }
    }
  }
}

```

System:
- Mamp Pro 5.5.1
- PHP 7.2.21
- Apache
- WP-GraphQL 0.3.6

Other plugins
- ac-addon-acf
- acf-cleaner-master
- acf-custom-database-tables
- admin-columns-pro
- advanced-custom-fields-pro
- classic-editor
- clean-image-filenames
- contact-form-7-multilingual
- disable-comments
- duplicator
- german-slugs
- sitepress-multilingual-cms
- wordpress-seo
- wp-graphiql-master
- wp-graphql-develop
- wp-media-library-categories
- wp-optimize
- wp-seo-multilingual
- wpai-acf-add-on
- wpml-media-translation

**Comment by ddresch** [comment]
see #892 which fixes this issue.

**Comment by jasonbahl** [comment]
Fixed in #1025

**Comment by florianschommertz** [comment]
Great Work!
